src/runtimes/node/sourcemap.js

```diff
@@ -0,0 +1,46 @@
+const { readFile, writeFile } = require('fs')
+const { dirname, relative, resolve } = require('path')
+const { promisify } = require('util')
+
+const pReadFile = promisify(readFile)
+const pWriteFile = promisify(writeFile)
+
+const PATH_FORMAT_ABSOLUTE = 'absolute'
+const PATH_FORMAT_RELATIVE = 'relative'
+
+const getPathFormat = (configProperty) =>
+  configProperty === PATH_FORMAT_ABSOLUTE ? PATH_FORMAT_ABSOLUTE : PATH_FORMAT_RELATIVE
+
+// Takes a path to a sourcemap file and rewrites it to the same location with
+// the paths in `sources` transformed based on the value of `pathFormat`.
+// - If set to `absolute`, absolute paths will be used;
+// - If set to `relative`, the paths will be relative to `srcDir`.
+const processSourcemap = async ({ bundlePath, pathFormat: pathFormatConfig, sourcemapPath, srcDir }) => {
+  if (!sourcemapPath) {
+    return
+  }
+
+  const bundleDir = dirname(bundlePath)
+  const pathFormat = getPathFormat(pathFormatConfig)
+  const data = await pReadFile(sourcemapPath, 'utf8')
+  const sourcemap = JSON.parse(data)
+  const newSources = sourcemap.sources.map((path) => {
```

Comment:
Is this required even when setting the right `sourceRoot` in the source map file?

Reply:
Oh no! This entire file is not needed anymore. I should've removed it in the last commit. I'm really sorry! 🤦🏻 

Done in https://github.com/netlify/zip-it-and-ship-it/pull/447/commits/68253759c76c0b07e1da1f73a932c76f6bd0a892.